draft-ietf-rats-architecture.md

```diff
@@ -1516,27 +1524,26 @@ The following example illustrates a hypothetical Background-Check Model solution
         |                |                |               |
         ~                ~                ~               ~
         |                |                |               |
-     time(HR_a)<---------+-------------time(HR_v)------>time(HR_r)
+     time(HR_a)<---------+-----------time(HR_v)----->time(HR_r)
         |                |                |               |
      time(EG_a)          |                |               |
-        |----Evidence{time(EG_a)}-------->|               |
-        | {Handle1,time(EG_a)-time(VG_a)}|                |
-        |                |             time(RG_v)         |
-        |<-----Attestation Result---------|               |
-        |   {time(RG_v),time(RX_v)}       |               |
-        |                |                                |
-        ~                ~                                ~
-        |                |                                |
-     time(HR_a')<--------'---------------------------->time(HR_r')
-        |                                                 |
-     time(RR_a)                                           /
-        |--Attestation Result{time(RX_v)-time(RG_v)}-->time(RA_r)
-        |    {Handle2, time(RR_a)-time(EG_a)}             |
-        ~                                                 ~
-        |                                                 |
-        |                                              time(OP_r)
-        |                                                 |
+        |---Evidence{H,time(VG_a)}------->|               |
```

Comment:
time(VG_a) makes no sense in this example since clocks are not synchronized so the actual value is meaningless, and there's no other time(foo_a) transmitted to the attester to use as a delta.
The old (red) text had time(EG_a)-time(VG_a) which did make sense, as the delta is meaningful to the receiver (it just requires a clock in the Attester that need not be synchronized).

That's why the old text had time(RX_v)-time(RG_v) is is meaningful.